WordPress/src/main/java/org/wordpress/android/ui/DeepLinkingIntentReceiverActivity.java

```diff
@@ -91,14 +96,29 @@ protected void onCreate(Bundle savedInstanceState) {
             } else if (shouldShowPages(uri)) {
                 handleShowPages(uri);
             } else {
-                // not handled
-                finish();
+                UriWrapper uriWrapper = new UriWrapper(uri);
+                // https://public-api.wordpress.com/bar/
+                if (mEmailUriHandler.shouldHandleUri(uriWrapper)) {
+                    mEmailUriHandler.handleUri(uriWrapper);
+                } else {
+                    // not handled
+                    finish();
+                }
```

Comment:
Again, if we decide to handle `mbar` links, I believe we should be able to merge this logic with the existing logic in `handleOpenEditorFromTrackingUrl` as they seem to do the similar things, right? Let me know what you think.